src/pages/Dashboard/MintRepayVai/MintVai/index.tsx

```diff
@@ -27,6 +28,35 @@ export interface IMintVaiUiProps {
   mintFeePercentage?: number;
 }
 
+const MintFee = ({ mintFeePercentage }: Pick<IMintVaiUiProps, 'mintFeePercentage'>) => {
+  const [{ value }] = useField('amount');
+  const styles = useStyles();
+  const { t } = useTranslation();
+  const getReadableMintFee = React.useCallback(
+    (valueWei: string) => {
+      if (!mintFeePercentage) {
+        return '-';
```

Comment:
I know that's my code, but now that we have a constant for this character can you update this line please?
```suggestion
        return PLACEHOLDER_KEY;
```